commands/host/remote-db

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+## Description: Grab a remote DB and import it to the current Drupal project.
+## Usage: remote-db
+## Example: "ddev remote-db"
+
+echo "================================"
+echo "=     I can't do that, Dave.   ="
+echo "================================"
+echo "= Since moving from Docksal to ="
+echo "= DDEV, we have removed this   ="
+echo "= custom command in favour of  ="
+echo "= the built-in 'ddev pull' one ="
+echo "=                              ="
+echo "= You should be able to type   ="
+echo "= 'ddev pull platform' to get  ="
+echo "= both files and DB for the    ="
+echo "= current project. More detail ="
+echo "= on the knowledge.            ="
+echo "= https://bit.ly/ddev-db       ="
```

Comment:
Up until now this is a public repository with nothing referencing non-public content. It seems better to point people to https://ddev.readthedocs.io/en/latest/users/providers/platform where they learn how to configure their local global config and also how pull works.

I like the explanatory command, it will help transitioning from Docksal.

Reply:
Depends on the audience we're looking to attract with this repo. 
If we're focusing on people outside of the Annertech development team, then this is a different tool/solution than I had imagined.